pkg/gateway/configuration_workingset.go

```diff
@@ -101,10 +123,6 @@ func (c *WorkingSetConfiguration) readOnce(ctx context.Context, dao db.DAO) (Con
 
 		serverName := server.Snapshot.Server.Name
 
-		if _, exists := servers[serverName]; exists {
```

Comment:
This was an erroneous check anyways. We already do this validation in a profile when it's saved.